recipes/short-read-connector/meta.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+  name: shortreadsconnector
+  version: '1.1.0'
+
+source:
+  fn: rconnector-v1.1.0-Source.tar.gz
+  url: "https://github.com/GATB/short_read_connector/releases/download/v1.1.0/rconnector-v1.1.0-Source.tar.gz"
+  md5: "822289b00e644842e5553eee8484fbc8"
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - gcc   # [linux]
+    - llvm  # [osx]
+    - cmake
+    - zlib {{CONDA_ZLIB}}*
+    - git
```

Comment:
why is git needed?

Reply:
Hi @bgruening. Thanks for the review.

Inside the INSTALL file, there are git commands to get external repositories but it seems the dependency is not needed :)